README.md

```diff
@@ -1256,6 +1256,674 @@ The type of cost
+to show how one can use the tracking function to track a marker with a body segment.
+
+In this case, we use the `ConstraintFcn.TRACK_MARKER_WITH_SEGMENT_AXIS` constraint function, as shown below in the 
+definition of the constraints of the problem:
+
+```python
+constraints = ConstraintList()
+constraints.add(
+ConstraintFcn.TRACK_MARKER_WITH_SEGMENT_AXIS, node=Node.ALL, marker_idx=1, segment_idx=2, axis=Axis.X
+)
+```
+
+Here, we minimize the distance between the marker with index 1 ans the x axis of the segment with index 2. We align 
+the axis toward the marker. 
+
+### The track_segment_on_rt.py file
+This example is a trivial example where a stick must keep its coordinate system of axes aligned with the one
+from a box during the whole duration of the movement. The initial and final position of the box are dictated,
+the rest is fully optimized. It is designed to show how one can use the tracking RT function to track
+any RT (for instance Inertial Measurement Unit [IMU]) with a body segment.
+
+To implement this tracking task, we use the `ConstraintFcn.TRACK_SEGMENT_WITH_CUSTOM_RT` constraint function, which 
+minimizes the distance between a segment and an RT. The extra parameters `segment_idx: int` and `rt_idx: int` must be 
+passed to the Objective constructor.
+
+## Moving estimation horizon
+In this section, we perform mhe on the pendulum example.
+
+### The mhe.py file
+In this example, we apply mhe (Moving Horizon Estimation) on a simple pendulum simulation. 
+To better understand this example, let’s take a look at the structure of the code. 
+
+First of all, the generate_data function generates states (X_), controls (U_) and marker trajectories (Y_ ) for our 
+pendulum example, using scipy.integrate.solve_ivp. It also returns noisy data for the marker trajectories (Y_N).
+
+The check_results function returns Y_est which corresponds to the estimated marker trajectories, based on the generated
+controls Xs (which is one of the arguments of the check_results function). 
+
+The `plot_true_X` function allows user to simply plot the generated states X_. 
+The `plot_true_U` function allows user to simply plot the generated controls U_. 
+
+The warm_start_mhe function is used to shift forward to the next frame, discarding the oldest frame (estimated controls 
+and states) of the window. At the same time, the function returns the discarded controls X_out. 
+
+In the prepare_ocp function, we define the model path, the dynamics, the initial guesses, the path constraints, the 
+control path constraints. We also add objective functions to the objective_functions list. 
+
+In the main, different parameters are defined.
+The duration of the simulation : `Tf = 5s`,
+the number of shooting nodes : `N = 500`,
+the STD of noise added to measurements : `noise_std = 0.05`,
+the max torque applied to the model : `T_max = 2`,
+the number of frames of the mhe window :` N_mhe = 10`.
+
```

Comment:
This is too precise, keep it to an abstract level

Reply:
I decided to completely change the structure of this paragraph, to keep it to an abstract level.

Reply:
Done